pydantic/schema.py

```diff
@@ -487,9 +489,12 @@ def field_type_schema(
             definitions.update(sf_definitions)
             nested_models.update(sf_nested_models)
             sub_schema.append(sf_schema)
-        if len(sub_schema) == 1:
-            sub_schema = sub_schema[0]  # type: ignore
-        f_schema = {'type': 'array', 'items': sub_schema}
+
+        f_schema = {'type': 'array', 'minItems': tuple_len, 'maxItems': tuple_len}
+        if tuple_len == 1:
+            f_schema['items'] = sub_schema[0]
```

Comment:
this looks weird, are you sure it's correct as per JSONSchema.

Surely even if there's only one item, `items` should be an array?

Reply:
It's valid and more common to directly put the type like [here](https://json-schema.org/understanding-json-schema/reference/array.html#list-validation)
```json
{
  "type": "array",
  "items": {
    "type": "number"
  }
}
```
when the type is the same for all fields (which is the case if there is only one expected item of course)
But both are valid I don't have a strong opinion on this